src/writeData/components/PluginCreateConfiguration/Footer.tsx

```diff
@@ -163,7 +162,7 @@ const mstp = (state: AppState) => {
       steps: {orgID},
     },
   } = state
-  const accountType = getQuartzMe(state)?.accountType ?? 'free'
+  const accountType = state?.identity?.currentIdentity?.account?.type ?? 'free'
```

Comment:
I've changed the logic here because I do not think there should be a need to directly call the selector function.

Either we call the selector function using `useSelector` in the React component, or we directly access the `state` object passed into `matchStateToProps` as an argument.

Reply:
This is not a cloud-only component. `accountType` will be empty string by default. 

As before, there is no need for an additional `CLOUD` guard because `accountType` is already guarded by, and only used, within a `CLOUD` check on line 69.

```
  if (CLOUD && accountType === 'free') {
        config = config.replace(
          agentSettingPatternDefault,
          agentSettingPatternFreeTier
        )
      }
```

Reply:
You're right on both counts - let me revise this and I'll rebase and push up again.